modules/supported.py

```diff
@@ -0,0 +1,116 @@
+"""
+Support for os and other modules which written for avoiding repetitive code.
+"""
+import os
+from modules import run_command, constants
+
+NAME_FIELD = 'NAME='
+VERSION_FIELD = 'VERSION_ID='
+ALPINE = 'alpine'
+
+
+def file_content(file_path, debug, container_name):
+    """This function checks returns the file's content if exists."""
+    content = ''
```

Comment:
should probably be a list
```suggestion
    content = []
```